src/server_manager/web_app/app.ts

```diff
@@ -620,8 +621,13 @@ export class App {
       return;
     }
 
-    await this.loadGcpAccount(gcpAccount);
-    this.showIntro();
+    const gcpServers = await this.loadGcpAccount(gcpAccount);
+    if (gcpServers.length > 1) {
```

Comment:
I think you meant this:
```suggestion
    if (gcpServers.length > 0) {
```